src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.cs

```diff
@@ -590,7 +609,7 @@ private T IntroduceFrame<T>(BoundNode node, LambdaFrame frame, Func<ArrayBuilder
 
             var result = F(prologue, addedLocals);
 
-            _framePointers.Remove(frame);
+            //_framePointers.Remove(frame);
```

Comment:
>// [](start = 12, length = 2)

Can you please expand this comment slightly into an explanation of why we don't do it?